exporter/terra/transfer.py

```diff
@@ -14,6 +16,17 @@ def __init__(self, gcs_transfer: GcsTransfer, aws_access_key_id: str, aws_access
         self.gcs_bucket_prefix = gcs_dest_prefix
         self.notification_topic = notification_topic
 
+    @staticmethod
+    def from_env():
+        aws_access_key_id = os.environ['AWS_ACCESS_KEY_ID']
```

Comment:
Do we want it to raise an exception if the value does not exist? 

Probably yes, but I wanted to double check! If not, we probably want to us `os.getenv()`